airbyte-ci/connectors/pipelines/pipelines/dagger/actions/connector/hooks.py

```diff
@@ -3,16 +3,23 @@
 #
 
 import importlib.util
+from importlib import metadata
 from importlib.abc import Loader
 
 from dagger import Container
-from dagger.engine._version import CLI_VERSION as dagger_engine_version
 from pipelines.airbyte_ci.connectors.context import ConnectorContext
 
 
+def get_sdk_version() -> str:
```

Comment:
Not immediately clear what SDK its refering to when you call it.
```suggestion
def get_dagger_sdk_version() -> str:
```

Reply:
Done